rules/docker-fpm-frr.mk

```diff
@@ -30,6 +30,8 @@ SONIC_DOCKER_DBG_IMAGES += $(DOCKER_FPM_FRR_DBG)
 $(DOCKER_FPM_FRR)_CONTAINER_NAME = bgp
 $(DOCKER_FPM_FRR)_RUN_OPT += --privileged -t
 $(DOCKER_FPM_FRR)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
+$(DOCKER_FPM_FRR)_RUN_OPT += -v /etc/timezone:/etc/timezone:ro
+$(DOCKER_FPM_FRR)_RUN_OPT += -v /etc/localtime:/etc/localtime:ro
```

Comment:
this is the symbol link, how does mount work here?

Reply:
> the problem seems not specific to frr, why other containers do not need this? @qiluo-msft , do you have any concern for mount them into the container?

This issue persist across all the containers. We can use the same technique to sync time/timezone on other containers too.

Reply:
> this is the symbol link, how does mount work here?

Yes "/etc/localtime" is a symlink and it works only when we restart the BGP container. For instance, if we have changed the timezone on the host and to sync that with the container, we need to restart the container to update the respective mount share inside the container. With this, we don't need to mount the absolute path which is "/usr/share/zoneinfo".

Reply:
@nmoray Mounting a symlink on host without mounting the linked file at the same time is problematic. And this symlink is distro implementation detail, and it may break if any host/container distro change.

I am trying to understand your motivation at the very beginning. If your goal is to customize timezone used in BGP container, the best fix it to change timezone in BGP container only. Why you need to change timezone on the host and expect the FRR and SONiC time sync? The time sync is a nice-to-have feature, but overkill for your need.

Reply:
@qiluo-msft I agree that we should mount both symlink along with the target file. But why do you think the distro change will break the implementation? If the container distro changes, it will automatically update the mounted symlink too, right? 

Yes, we can manually update the timezone in each and every container. But the motive behind doing this is the auto time / zone sync between the container(s) and host.